tests/virtual_machines_test.go

```diff
@@ -59,10 +59,8 @@ var _ = Describe("Virtual Machines", func() {
 				vm := CreateVmObject(TestNamespace, false, []kubevirtv1.Interface{newInterface("br", "")},
 					[]kubevirtv1.Network{newNetwork("br")})
 
-				Eventually(func() error {
-					return testClient.VirtClient.Create(context.TODO(), vm)
-
-				}, timeout, pollingInterval).ShouldNot(HaveOccurred(), "failed to apply the new vm object")
+				err = testClient.VirtClient.Create(context.TODO(), vm)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
@alonSadan, since these changes are not really for this PR and we don't have really time to tackle those now. Would you please open an Issue with your suggestions, so we don't forget about it? Thanks

Reply:
Done